### PR TITLE
feat(frontend): Redirect noncomplaint 2FA users to the 2FA settings page

### PR DESCRIPTION
**Pull request by lauryndbrown**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationContext.jsx [hunk 1]
```diff
@@ -178,6 +180,17 @@ const OrganizationContext = createReactClass({
               </div>
             </div>
           );
+        case ERROR_TYPES.ORG_REQUIRES_2FA:
+          window.location.href = '/account/settings/2fa/' + '?is_not_2fa_compliant=True';
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/organizationContext.jsx]: this probably doesn't need to be concatted

File: src/sentry/static/sentry/app/views/organizationContext.jsx [hunk 2]
```diff
@@ -178,6 +180,17 @@ const OrganizationContext = createReactClass({
...
+          return (
+            <div className="container">
+              <div className="alert alert-block">
+                {t(
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/organizationContext.jsx]: thoughts on putting a button here with `ExternalLink` so they can read before clicking?

**Review by MaxBittker — APPROVED** [review]
looks good to me, left some questions and nits! 

sorry it took so long, feel free to ping me in the future 
(also worth removing the WIP tag if things are ready for review)

**Comment by benvinegar** [comment]
@lauryndbrown and I discovered this PR isn't necessary because [the code here](https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/app.jsx#L108) already handles 401s and redirects to the appropriate location.

**Pull request closed without merging**